src/javascript/binary/pages/portfiliows.js

```diff
@@ -0,0 +1,181 @@
+var PortfolioWS =  (function() {
+
+    'use strict';
+
+    var pageLanguage = page.language();
+    if(!pageLanguage) pageLanguage = "EN";
+
+    var rowTemplate;
+    var retry;
+
+    var init = function() {
+        // get the row template and then discard the node as it has served its purpose
+        rowTemplate = $("#portfolio-dynamic tr:first")[0].outerHTML;
+        $("#portfolio-dynamic tr:first").remove();
+        TradeSocket.init({"thendo":"updateBalance"});
```

Comment:
Authorize does return a balance. Thought it would be better to use the data we have rather than requesting it again.